website/docs/guides/cloud_integrations_guide.html.markdown

```diff
@@ -20,7 +20,7 @@ If you encounter issues or bugs, please [report those on Github repository](http
 
 ### AWS
 
-The New Relic AWS integration relies on two mechanisms to get data in New Relic: [AWS Metric stream](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/connect/aws-metric-stream/) and [AWS Polling integrations](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/connect/connect-aws-new-relic-infrastructure-monitoring). For the majority of AWS services the AWS Metric stream is used as it [has many advantages compared to polling](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/connect/aws-metric-stream#why-it-matters). The AWS Polling integrations are also enabled because AWS does not yet [support all metrics through AWS Metric Stream](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/connect/aws-metric-stream#integrations-not-replaced-streams). We enable `AWS Billing`, `AWS CloudTrail`, `AWS Health`, `AWS Trusted Advisor`, `AWS X-Ray`, `AWS S3`, `AWS Doc DB`, `AWS SQS`, `AWS EBS`, `AWS ALB`, `AWS Elasticache`, `AWS ApiGateway`, `AWS AutoScaling`, `AWS AppSync`, `AWS Athena`, `AWS Cognito`, `AWS Connect`, `AWS DirectConnect` and `AWS Fsx`.
+The New Relic AWS integration relies on two mechanisms to get data in New Relic: [AWS Metric stream](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/connect/aws-metric-stream/) and [AWS Polling integrations](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/connect/connect-aws-new-relic-infrastructure-monitoring). For the majority of AWS services the AWS Metric stream is used as it [has many advantages compared to polling](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/connect/aws-metric-stream#why-it-matters). The AWS Polling integrations are also enabled because AWS does not yet [support all metrics through AWS Metric Stream](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/connect/aws-metric-stream#integrations-not-replaced-streams). We enable `AWS Billing`, `AWS CloudTrail`, `AWS Health`, `AWS Trusted Advisor`, `AWS X-Ray`, `AWS S3`, `AWS Doc DB`, `AWS SQS`, `AWS EBS`, `AWS ALB`, `AWS Elasticache`, `AWS ApiGateway`, `AWS AutoScaling`, `AWS AppSync`, `AWS Athena`, `AWS Cognito`, `AWS Connect`, `AWS DirectConnect`, `AWS Fsx`, `AWS Glue`, `AWS Kinesis Analytics`, `AWS MediaConvert`, `AWS Media Package vod`, `AWS Mq`, `AWS Msk`, `AWS Neptune`, `AWS Qldb`, `AWS Route53resolver`, `AWS States`, `AWS TransitGateway`, `AWS Waf`, `AWS Wafv2`, `Cloudfront`.
```

Comment:
Please consider incorporating these suggestions after including the related example.

Reply:
Done.